docs/develop/module-specifications/spec-governance.mdx

```diff
@@ -8,25 +8,34 @@ Terra's Governance module inherits from Cosmos SDK's [`gov`](https://docs.cosmos
 
 </Admonition>
 
-Governance is the process through which members of the Terra community can effect change on the protocol by submitting petitions known as "proposals" and arriving at a popular consensus when a threshold amount of support has been reached. The proposal structure is versatile and allows for holders of staked Luna (those who have an interest in the long-term viability of the network) to voice their opinion on both blockchain parameter updates as well as the future development of the Terra protocol.
+The Terra protocol is a decentralized public [blockchain](../../learn/glossary.mdx#blockchain) governed by community members. Governance is the democratic process that allows users and validators to make changes to the Terra protocol. Community members submit, vote, and implement proposals.
 
-Check the [Governance section of the Terrad Reference](../terrad/subcommands.mdx#tx-gov-submit-proposal) to see examples of how to participate in the Governance process.
+The governance module maintains the logic for all governance activity. 
 
-To learn how to vote with your staked Luna or submit proposals, visit the [Terra Station governance guide](../../learn/terra-station/governance.mdx).
+To learn how to vote with your staked Luna or submit proposals, visit the [Terra Station governance guide](../../learn/terra-station/governance.mdx). 
+
+If you are an advanced user and want to learn about submitting or voting on proposals using Terrad, visit the [Governance section of the Terrad Reference](../terrad/subcommands.mdx#tx-gov-submit-proposal). 
 
 ## Concepts
 
-The following is the governance proposal procedure:
+The following concepts describe the governance proposal procedure.
+
+### Drafting and submission
+
+Proposals start as ideas within the community on Terra's [Agora forum](https://agora.terra.money/). After gaining support and feedback from the community, a proposer [drafts and submits a proposal](../../learn/terra-station/governance.mdx#create-a-new-proposal) alongside an [initial deposit](../../learn/terra-station/governance.mdx#deposit-luna-to-a-proposal). 
+
+To learn more about the different types of proposals, see the [Proposal types](#proposal-types) section below. 
 
-### Deposit Period
+
+### Deposit period
 
 After a proposal is submitted, it enters the deposit period, where it must reach a total minimum deposit of 512 Luna within 7 days from the time of its submission. The deposit threshold is reached when the sum of the initial deposit (from the proposer) and the deposits from all other interested network participants meet or exceed 512 Luna.
 
 Deposits protect against unnecessary proposals and spam.
 
 Deposits get refunded if all of the following conditions are met:
 - The minimum deposit of 512 Luna is reached within the 7-day deposit period.
-- `Quorum` is met: the number of total votes is greater than 10% of all staked Luna
+- `Quorum` is met: the number of total votes is greater than 10% of all staked Luna.
```

Comment:
What do you think about:
"all casted votes represent more than 10% of all staked Luna."

Reply:
Good suggestion!

Reply:
Implementing.